### PR TITLE
Loot tracker - Use alchemy values for prices

### LINKED ISSUE #5813: Loot Tracker: Add toggle to show High Alch price instead of GE
**Is your feature request related to a problem? Please describe.**
Some players don't particularly care about GE price (ie. ironmen).

### PR DESCRIPTION
**Pull request by psikoi**
Fixes #5813

This adds a toggle for using alchemy value as a price for the loot tracker, this can be useful for ironmen and accounts of the sort.

Depends on #5830 and #5483

### CODE REVIEW

**Comment by Gamer1120** [comment]
Could you remove the dot at the end of the config option? It seems inconsistent with the other config options.

File: runelite-client/src/main/java/net/runelite/client/game/ItemManager.java [hunk 1]
```diff
@@ -204,6 +207,28 @@ public int getItemPrice(int itemID)
...
+		{
+			return 1000;
+		}
+
+		float alchValue = getItemComposition(itemID).getPrice() * HIGH_ALCHEMY_CONSTANT;
+
+		/*
+		 * If the alch value is below 1, do not round up, return 0 instead.
+		 * For example bones have a base price of 1, multiplied by 0.6 will be 0.6,
+		 * if rounded up, the alch price will be 1, when it should be 0.
```
Comment (ollieatkinson) [inline runelite-client/src/main/java/net/runelite/client/game/ItemManager.java]: Can you use `Math.floor` here? Saves the comment and the ternary

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	@ConfigItem(
+		keyName = "ignoredItems",
+		name = "",
+		description = ""
+	)
+	void setIgnoredItems(String key);
+
+	@ConfigItem(
+		keyName = "showAlchPrice",
+		name = "Show prices as alchemy values.",
```
Comment (ollieatkinson) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java]: Can you remove the full stop?

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+		description = "Configures whether or not to display the alchemy value of an item instead of it's GE price."
+	)
+	default boolean showAlchPrice()
+	{
+		return true;
```
Comment (ollieatkinson) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java]: Are we OK with changing the default behaviour? I think we should - but need to make sure it's a new intended default option for everyone

**Comment by ollieatkinson** [comment]
I like this, thanks for the hard work to get it done 👍

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java (hunk 3 above)
Comment (psikoi) [reply runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java]: Ah, my bad.

**Comment by cohen990** [comment]
Hey do we know why this was closed? I still would like this feature

**Pull request closed without merging**